rotkehlchen/chain/ethereum/modules/thegraph/decoder.py

```diff
@@ -108,6 +108,7 @@ def _decode_stake_delegated(self, context: DecoderContext) -> DecodingOutput:
                 event.balance = Balance(amount=stake_amount_norm)
                 event.notes = f'Delegate {stake_amount_norm} GRT to indexer {indexer}'
                 event.counterparty = CPT_THEGRAPH
+                event.extra_data = {'indexer': indexer}
```

Comment:
you will need to update the test to add this field for the decoded transactions. Don't forget to do it